### DISCUSSION TITLE
setting line thickness and colour for geojson

### DISCUSSION THREAD

**Discussion by mattpgreen** [TerriaJS/terriajs#7260] [General] [answered]
hi, is there a way to set the line thickness and colour for geojson items - currently it appears to default to 1px which doesn't stand out from the background

**Comment by zoran995** [comment, marked as answer]
Hi @mattpgreen, sorry for the late answer. The easiest way to style geojson features in terria is to use advanced style editor. 
Here is a screen recording of how you can achieve changing the line width (ignore the copy error it is because the clipboard can only be used with https)
https://github.com/user-attachments/assets/f18db5f2-8354-41ab-bbe3-3dc70e86a01b

I used https://gist.github.com/zoran995/d3767b3fc3ddee6bf39b7e1f9102b559 as a sample data set, and this is a sample style definition 
```
{"styles":[{"id":"User Style","color":{"colorColumn":"name"},"outline":{"null":{"width":5}},"hidden":false}],"activeStyle":"User Style"}
```
You just need to merge this style definition in catalog item JSON
